package.json

```diff
@@ -33,7 +33,8 @@
     "test:unit": "vitest",
     "generate-changelog": "source ghtoken; GITHUB_TOKEN=${GITHUB_TOKEN} changeset version",
     "lint": "prettier --plugin-search-dir . --check src && eslint .",
-    "format": "prettier --plugin-search-dir . --write src"
+    "format": "prettier --plugin-search-dir . --write src",
+    "prepare":"npm build"
```

Comment:
I'm fairly sure this needs to be `npm run package`.

To test this from another repo, specify a dep like `"svelte-maplibre": "github:stuartlynn/svelte-maplibre#prepare_script"`. That made me see that `npm build` needs to at least be `npm run build`. But then I get errors involving the `.env` variable `PUBLIC_MAPTILER_KEY`. Checking through the existing scripts, looks like `package` does the trick. Seems to be working locally -- `node_modules/svelte-maplibre/dist` exists as expected, after a slightly lengthy build step

Reply:
Yeah that's right. The `build` command actually just builds the demo app. `package` builds the code that you would actually want when using this project as a dependency.

Reply:
Ack sorry my bad. Was in a rush and wanted to get this in quickly. Your both right, package is what this should be.